src/gui/dialogs/game_load.cpp

```diff
@@ -369,6 +369,22 @@ void game_load::evaluate_summary_string(std::stringstream& str, const config& cf
 	if(!cfg_summary["version"].empty()) {
 		str << "\n" << _("Version: ") << cfg_summary["version"];
 	}
+
+	const std::vector<std::string>& active_mods = utils::split(cfg_summary["active_mods"]);
+	if (!active_mods.empty()) {
+		str << "\n" << _("Modifications: ");
+		for (const auto& mod_id : active_mods) {
+			std::string mod_name;
+			try {
+				mod_name = cache_config_.find_child("modification", "id", mod_id)["name"].str();
```

Comment:
`.str()` should be unnecessary, attribute values have implicit conversion to strings.

Reply:
Without the `.str()` it doesn't compile.

```
src/gui/dialogs/game_load.cpp:382:77: error: ambiguous overload for ‘operator=’ (operand types are ‘std::__cxx11::string {aka std::__cxx11::basic_string<char>}’ and ‘const attribute_value {aka const config_attribute_value}’)
     mod_name = cache_config_.find_child("modification", "id", mod_id)["name"];
```